specification/recoveryservicesdatareplication/resource-manager/readme.java.md

```diff
@@ -0,0 +1,12 @@
+## Java
+
+These settings apply only when `--java` is specified on the command line.
+
+``` yaml $(java)
+title: DataReplicationMgmtClient
+service-name: Recovery Services Data Replication
+
+graal-vm-config: false
+remove-operation-group: ReplicationExtensionOperationStatus
```

Comment:
why do we remove this operation grourp?

Reply:
1st, name too long
2nd, it just the operation group for LRO polling status endpoint (we can add later if needed)